frontend/src/locales/sql-review/en-US.json

```diff
@@ -47,7 +47,7 @@
           "title": "Table name format (regex)"
         },
         "maxLength": {
-          "title": "Length limit"
+          "title": "Length limit (not support PostgreSQL)"
```

Comment:
Sounds a little bit weird. What about "PostgreSQL not supported"?

Reply:
Good for me, updated